poster/poster.tex

```diff
@@ -218,12 +220,15 @@ \section*{BackgroundSub}
 
 \para
 Finally, if desired, the calculated background can be subtracted off
```

Comment:
Rewrote the language to talk about the "in-memory cache" rather than an image "falling out of memory"; the latter (to me) sounds like an accident ("Aunt Sally fell out the attic window"!) :smile_cat:

Reply:
"Down will come baby, cradle and all!"